Software/real_time/ROS_RoboBuggy/src/robobuggy/scripts/basic_gps_subscriber.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env/python
```

Comment:
What is this file for?

Reply:
that was just meant to be for learning about ROS along with the rest of the team; I put it on my own branch but forgot to take it out

Reply:
resolved